ehsm_kms_service/function.js

```diff
@@ -37,13 +37,17 @@ const base64_decode = (base64_str) =>
 
 /**

```

Comment:
can you add more explanation for the issue and the fix solution in the commit msg?

Reply:
ok